test/model/binding.dart

```diff
@@ -560,8 +560,8 @@ class FakeAndroidNotificationHostApi implements AndroidNotificationHostApi {
 
   final Map<String, MessagingStyle?> _activeNotificationsMessagingStyle = {};
 
-  /// Clears all active notifications that have been created via [notify].
-  void clearActiveNotifications() {
+  /// Clears all messaginging style of active notifications that have been created via [notify].
```

Comment:
nit: spelling; also, instead of "messaging style", maybe this should say `[MessagingStyle]`? It's not intuitively clear what a "messaging style" is, and maybe Android could have chosen a better name for it, but using the identifier should at least shorten the reader's path to looking up what it means in the Android documentation.